File: erubit.platform/src/main/java/a/erubit/platform/android/PreferencesFragment.java
package a.erubit.platform.android;


import android.os.Bundle;
import android.support.v4.content.ContextCompat;
import android.support.v7.preference.PreferenceFragmentCompat;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import a.erubit.platform.R;

public class PreferencesFragment extends PreferenceFragmentCompat implements IUxController {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        addPreferencesFromResource(R.xml.preferences);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = super.onCreateView(inflater, container, savedInstanceState);
        if (view != null)
            view.setBackgroundColor(ContextCompat.getColor(getContext(), R.color.color_bg_light));

        return view;
    }

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
    }

    @Override
    public int getFloatingButtonVisibility() {
        return View.GONE;
    }
}

File: erubit.platform/src/main/java/a/erubit/platform/android/PreferencesFragment.kt
package a.erubit.platform.android

import a.erubit.platform.R
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.core.content.ContextCompat
import androidx.preference.PreferenceFragmentCompat

class PreferencesFragment : PreferenceFragmentCompat(), IUxController {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)

		addPreferencesFromResource(R.xml.preferences)
	}

	override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
		val view = super.onCreateView(inflater, container, savedInstanceState)

		view?.setBackgroundColor(ContextCompat.getColor(context!!, R.color.color_bg_light))

		return view
	}

	override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {}

	override val floatingButtonVisibility: Int
		get() = View.GONE
}
